tmail-backend/jmap/extensions/src/main/scala/com/linagora/tmail/james/jmap/method/EmailRecoveryActionSetMethod.scala

```diff
@@ -52,16 +57,26 @@ class EmailRecoveryActionMethodModule extends AbstractModule {
 object EmailRecoveryActionConfiguration {
 
   val DEFAULT_MAX_EMAIL_RECOVERY_PER_REQUEST: Long = 5
+  val DEFAULT_RESTORATION_HORIZON: Duration = DurationParser.parse("15", ChronoUnit.DAYS)
 
   def from(propertiesProvider: PropertiesProvider): EmailRecoveryActionConfiguration = {
-    val maxEmailRecoveryPerRequest: Option[Long] = Try(propertiesProvider.getConfiguration("jmap"))
-      .map(configuration => configuration.getLong("emailRecoveryAction.maxEmailRecoveryPerRequest"))
+    val config = Try(propertiesProvider.getConfiguration("jmap"))
+
+    val maxEmailRecoveryPerRequest: Option[Long] = config
+      .map(_.getLong("emailRecoveryAction.maxEmailRecoveryPerRequest"))
       .toOption
-    EmailRecoveryActionConfiguration(maxEmailRecoveryPerRequest = maxEmailRecoveryPerRequest.getOrElse(DEFAULT_MAX_EMAIL_RECOVERY_PER_REQUEST))
+    val restorationHorizon: Option[Duration] = config
+      .map(_.getString("emailRecoveryAction.restorationHorizon"))
```

Comment:
We now need to document this new property:

https://github.com/linagora/tmail-backend/blob/master/docs/modules/ROOT/pages/tmail-backend/configure/index.adoc#additional-jmap-configurations